### PR TITLE
Daniel to dev

### PR DESCRIPTION
**Pull request by YoungMasterGandalf**
This PR is linked to:
* [https://github.com/ForloopAI/forloop_modules/pull/36](https://github.com/ForloopAI/forloop_modules/pull/36)
* [https://github.com/ForloopAI/forloop_cloud/pull/217](https://github.com/ForloopAI/forloop_cloud/pull/217)

It brings:
* Refactor `LoadPythonScript` and `LoadJupyterScript` handlers
* implement new model for load script API endpoint

### CODE REVIEW

**Review by DovaX — CHANGES_REQUESTED** [review]
I dont understand where you backed up the code for reading of the jupyter notebook :D I didnt want you to deprecate it 👍 
+ also it creates scripts, but where is a node to run script?

**Comment by YoungMasterGandalf** [comment]
> I dont understand where you backed up the code for reading of the jupyter notebook :D I didnt want you to deprecate it 👍
> 
> * also it creates scripts, but where is a node to run script?

Do you mean the commented one, the "try - except - finally" part? I thought it was obsolete so I didn't back it up :D

**Comment by YoungMasterGandalf** [comment]
> I dont understand where you backed up the code for reading of the jupyter notebook :D I didnt want you to deprecate it 👍
> 
> * also it creates scripts, but where is a node to run script?

I didn't finish the running scripts part yet. The DB tasks seemed like a greater priority so I wanted to push this, go to them and return to running scripts @DovaX

File: forloop_modules/queries/db_model_templates.py [hunk 1]
```diff
@@ -201,9 +201,14 @@ class APIFile(BaseModel):
 
 class APIVariableToBeAssignedToFile(BaseModel):
     variable_name: str = ""
+    node_uid: str = ""
     project_uid: str = "0"
     pipeline_uid: str = "0"
+    
 
+class APILoadScriptViaNode(BaseModel):
+    node_uid: str = ""
```
Comment (DovaX) [inline forloop_modules/queries/db_model_templates.py:210]: node_uid = "0" is maybe better? -> then, if 0 is passed anywhere - it means you didnt specify it where you should do that :)

**Review by DovaX — APPROVED** [review]

**Pull request merged**